doc/src/devdocs/gc.md

```diff
@@ -59,3 +59,45 @@ GC heuristics tune the GC by changing the size of the allocation interval betwee
 The GC heuristics measure how big the heap size is after a collection and set the next collection according to the algorithm described by https://dl.acm.org/doi/10.1145/3563323, in summary, it argues that the heap target should have a square root relationship with the live heap, and that it should also be scaled by how fast the GC is freeing objects and how fast the mutators are allocating. The heuristics measure the heap size by counting the number of pages that are in use and the objects that use malloc. Previously we measured the heap size by counting the alive objects, but that doesn't take into account fragmentation which could lead to bad decisions, that also meant that we used thread local information (allocations) to make decisions about a process wide (when to GC), measuring pages means the decision is global.
 
 The GC will do full collections when the heap size reaches 80% of the maximum allowed size.
+
+## Julia + MMTk
```

Comment:
I feel we may want to separate this into a different file. Probably check with @d-netto and others.

Reply:
sure